src/examples/components/button.tsx

```diff
@@ -29,7 +29,7 @@ export function Button(origProps: ButtonProps) {
         props.active && styles.active,
         props.reversed && styles.reversed,
       )}
-      onMouseDown={() => props.onMouseDown}
+      onMouseDown={props?.onMouseDown}
```

Comment:
Ah yeah, that was my bad. I had made the first change due to linting complaining. I actually fixed by removing onMouseDown from splitProps, but looks like I forgot to merge. If you see this line disappear, that's why.